### PR TITLE
[Review] Some Minor Stuff

### PR DESCRIPTION
**Pull request by ombre8**
Thanks for the Lab it is fun, I found some minor stuff I would change:

- Added a Note about provided VMs and IPadresses see: #21
- Added hint about checking metrics first and drop it then
- changed job to prometheus since node_exporter does not have dropped
  metric
- changed titels in Solutions to bold so they do not show up in side
  navigation and spoil all the fun
- added a note about staleness
- streamlined format of alertmanager components to same as used in chapter 1

### CODE REVIEW

File: content/en/docs/01/labs/11.md [hunk 1]
```diff
@@ -25,8 +26,9 @@ tar fvxz node_exporter-1.0.1.linux-amd64.tar.gz
 ```
 
 The Prometheus configuration part can be solved by either using a static configuration **or** by using the file service discovery mechanism:
-
-### Option 1: Static config
+<!-- markdownlint-disable MD036 -->
```
Comment (phil-pona) [inline content/en/docs/01/labs/11.md:29]: I wouldn't use those

Comment (ombre8) [reply content/en/docs/01/labs/11.md:29]: the alternative is to disable the right Navigationbar or to use 4. Level Headings which will trigger MD001 and maybe others... Or do not use Headings in Solutions at all

Comment (phil-pona) [reply content/en/docs/01/labs/11.md:29]: I see, let me quickly have a look at it.

Comment (phil-pona) [reply content/en/docs/01/labs/11.md:29]: I'd rather find general solution (we might need this in other trainings as well) to disable the right navigation for solutions, than disabling the linting. It adds unnecessary boilerplate code and it's quite a hacky solution.

Comment (ombre8) [reply content/en/docs/01/labs/11.md:29]: let me know if you have such a solution and I will adapt this MR

Comment (phil-pona) [reply content/en/docs/01/labs/11.md:29]: Are you ok with it, if we switch back to h3 and merge, for tomorrows session, to not have a longliving PR? then back back to it, once we have a general solution?

**Review by phil-pona — APPROVED** [review]

File: content/en/docs/01/labs/11.md (hunk 1 above)
Comment (ombre8) [reply content/en/docs/01/labs/11.md]: sry, missed your last comment, thanks for the change

**Pull request merged**